pkg/validations/filetests/one_of=/invalid-when-value-not-in-string

```diff
@@ -0,0 +1,7 @@
+#@assert/validate one_of="debug,info,warning,error,fatal"
+foo: critical
```

Comment:
this seems a bit suspicious. should we be disallowing string in string check? e.g. bad setup:

```
#@assert/validate one_of="not-valid"
foo: valid
```

Reply:
Yeah, bad.  Cool; will pull back "string in string" support.  Simplifies the impl too. 👍🏻